.travis.yml

```diff
@@ -2,7 +2,9 @@ language: python
 python:
 - '2.7'
```

Comment:
I *think* it should be fine, python-coveralls supports 3.4. easy way to test though with travis/local tox